orchagent/pfcwdorch.cpp

```diff
@@ -461,9 +471,43 @@ void PfcWdSwOrch<DropHandler, ForwardHandler>::registerInWdDb(const Port& port,
     }
 }
 
+template <typename DropHandler, typename ForwardHandler>
+string PfcWdSwOrch<DropHandler, ForwardHandler>::getLosslessStr(string str, set<uint8_t>& losslessTc)
+{
+    SWSS_LOG_ENTER();
+
+    istringstream is(str);
+    string curStr;
+    string res;
+
+    while (getline(is, curStr, ',' ))
```

Comment:
>  [](start = 34, length = 1)

please remove the space here. make sure you check you code style before submit the pr.